AI/VCAI/AIhelper.h

```diff
@@ -17,14 +17,15 @@
 #include "ResourceManager.h"
 #include "BuildingManager.h"
 #include "ArmyManager.h"
+#include "HeroManager.h"
 #include "Pathfinding/PathfindingManager.h"
 
 class ResourceManager;
 class BuildingManager;
 
 
 //indirection interface for various modules
-class DLL_EXPORT AIhelper : public IResourceManager, public IBuildingManager, public IPathfindingManager, public IArmyManager
+class DLL_EXPORT AIhelper : public IResourceManager, public IBuildingManager, public IPathfindingManager, public IArmyManager, public IHeroManager
```

Comment:
We should do smth with this, `AIhelper` started inheriting everything. May be just change to public getters

Reply:
As soon as I am able to build reliable logic on top of VCAI I will start cutting old architecture. I am not 100% sure but I expect to get rid of this class and use managers directly. Finally I may end up having a separate AI project so that we are able to select AI in game launcher. Now it is more important to build reliable logic which can play average map successfully.

Reply:
It's discussable whether it's better to have separate AI or "two AI's in one" with on/off toggle. Both approaches have their cons and pros. Cons for separate AI would be VCAI not benefitting directly from code improvements common for both AI's and your AI not benefitting from VCAI improvements made by other people.

Reply:
Agreed. But we can move towards some shared things. Or I can forever develop in my own branch and pack its result as mod. This way we can exchange commits using cherry pick or something. Anyway for now I do not plan to move to a separate assembly. It is lowest priority task.